README.md

```diff
@@ -195,5 +195,9 @@ Summary
 
 # Open Questions
 - If an extension package defines more than one TfM, the driver will just use the first one. Should this be configurable. Can't think of any scenarios for this right now.
+- If an extension is using a different framework than the project, can it still run?
```

Comment:
From a technical perspective, there shouldn't be any reason why not.  The extension has an independent package graph, and is executed in it's own process.
I see that this isn't spelled out clearly enough in the doc though.

Do you see a reason this should be prohibited?